doc/dlt_offline_logstorage.md

```diff
@@ -74,6 +74,7 @@ NOFiles=<number of files>            # Number of created files before oldest is
 SyncBehavior=<strategy>              # Specify sync strategy. Default: Sync'ed after every message. See Logstorage Rinbuffer Implementation below.
 EcuID=<ECUid>                        # Specify ECU identifier
 SpecificSize=<spec size in bytes>    # Store logs in storage devices after specific size is reached.
+GzipCompression=<0 or 1>             # Write the logfiles with gzip compression.
```

Comment:
Set the option to 0 causing error to read filter since dlt_logstorage_read_number will consider it is invalid:

```
[ 5543.704342]~DLT~ 3146~ERROR    ~Invalid, is not a number 
[ 5543.704343]~DLT~ 3146~ERROR    ~[ 5543.704344]~DLT~ 3146~WARNING  ~FILTER1 filter configuration is invalid 
[ 5543.704346]~DLT~ 3146~ERROR    ~dlt_daemon_logstorage_setup_internal_storage: Device connect failed
[ 5543.704347]~DLT~ 3146~INFO     ~Setting up internal offline log storage failed!
```
I highly suggest, for more comprehensive, it should be ON or OFF

Reply:
Agree. I queried about this particularly in my original PR submission. So we'll got with boolean config values as `on`/`off`. Or do we want something dynamic where `1`/`0` and `true`/`false` work in parallel with `on`/`off`?

Reply:
Fixed. I chose to document the "yes/no" values but the code will allow for 0/1 and true/false.

Reply:
Great. Thanks for the change. I think uppercase will look better and professional!? :) (ON/OFF).
I will test and comment later.

Reply:
Tested. Everything is fine now.
I just have 1 more question, since I don't have much experience with gzip. Will the logs be written to the zip file directly and in turn, or it waits for a number of logs (maybe based on file size), and then zip?
Thank you

Reply:
In regards to uppercase ("ON/OFF") we can change `strncmp` to `strncasecmp`. But I'm not 100% on the availability of the `<strings.h>` header on various platforms.

Reply:
Thank you,
For me, your current implementation is good now.
I need to check a few things related to performance (I am working on some stress tests, e.g. many apps sending many logs in short period of time delay, etc)
I will comment the result when done.